### PR TITLE
Kyle Teeter - Newsfeed Components

### PR DESCRIPTION
**Pull request by kyleteeter**
Project Components I

@Raymondgrc

### CODE REVIEW

File: Menu/Menu.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector(".menu");
+
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector(".menu-button");
+console.log(menuButton)
```
Comment (Raymondgrc) [inline Menu/Menu.js]: I like that you are making sure to test your code.

File: Article/Article.js [hunk 2]
```diff
@@ -3,24 +3,27 @@
...
+    this.expandButton.innerHTML = "expand";
     // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
+    this.expandButton.addEventListener('click', this.expandArticle)
     
   }
 
-  expandArticle() {
+  expandArticle(event) {
     // Using our reference to the article element, add or remove a class to expand or hide the article.
+    event.currentTarget.parentNode.classList.toggle('article-open');
```
Comment (Raymondgrc) [inline Article/Article.js]: That's a very long chain. It is apparent that you did your research.

Comment (kyleteeter) [reply Article/Article.js]: Thanks! I feel like I am getting a hang of chaining. It seems to make things a lot cleaner once I understand it.

**Comment by kyleteeter** [comment]
@Raymondgrc This might be a silly questions but which line 8 in which file were you referring to? (I can't tell from your comment. )

**Pull request closed without merging**